packages/shared/src/types/config/index.ts

```diff
@@ -27,14 +27,10 @@ export type RsbuildConfigMeta = {
 };
 
 /**
- * The shared Rsbuild config.
- * Can be used with both Rspack and Webpack.
+ * The rsbuild config to run in the specified environment.
  * */
-export interface RsbuildConfig {
-  /**
-   * Options for local development.
-   */
-  dev?: DevConfig;
+
+export interface RsbuildEnvironmentConfig {
```

Comment:
How about make it shorter:

```suggestion
export interface EnvironmentConfig {
```